src/common/vordr.ts

```diff
@@ -62,6 +83,20 @@ export const checkWithVordr = async (
     return true;
   }
 
+  // Check post title for vordr conditions
+  if (title && validatePostTitle(title)) {
+    logger.info(
+      { id, type, userId },
+      `Prevented ${type} because title contains banned content`,
+    );
```

Comment:
if we are logging, I think it would be useful to say what the title was and what it was set to, not much use of logs without that info.